src/state/selectors.ts

```diff
@@ -138,6 +140,35 @@ export function selectActiveCoordinate(
   } else {
     activeCoordinate = tooltipState.axisInteraction.activeClickCoordinate;
   }
+
+  // the following works for bar, area, line, but does not work for polar or scatter charts :(
```

Comment:
Is this inherent to Scatter and polar or is it because of the axis/item tooltip type?

Reply:
Everything else for axisType is being selected correctly I think?

I can use axis type to check things, but that y coordinate still doesn't exist in the data. 

I might need to reset some state 💡